src/infrastructure/BaseService.ts

```diff
@@ -27,6 +27,6 @@ export class BaseService {
     else if (token) this.headers['private-token'] = token;
 
     // Set sudo
-    if (sudo) this.headers['Sudo'] = sudo;
+    if (sudo) this.headers['Sudo'] = `${sudo}`;
```

Comment:
Why this change?

Reply:
Since `headers` value from `import { Options } from "ky";` are `string`.